README.md

```diff
@@ -48,6 +48,20 @@ At this point you will have the git local repository of the `snappy` source at
 `$GOPATH/github.com/ubuntu-core/snappy/snappy`. The source for any
 dependent packages will also be available inside `$GOPATH`.
 
+### Dependencies handling
```

Comment:
In theory, you should have had all the required dependencies as it is stated above to run `go get` which will traverse the source and grab everything.

Maybe it is better to suggest for `HACKING` to `git clone` to the appropriate path instead with a subsequent run for dependency handling.